package/butterfly_angular/lib/butterfly_angular.dart

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2017, jonahwilliams. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+/// Support for placing Angular components in a Butterfly application.
+library butterfly_angular;
+
+import 'dart:html' as html;
+
+import 'package:angular/angular.dart';
+import 'package:angular/src/core/application_ref.dart';
+import 'package:angular/src/di/injector/hierarchical.dart';
+import 'package:angular/src/di/injector/injector.dart';
+import 'package:angular/src/platform/dom/events/event_manager.dart';
+import 'package:butterfly/butterfly.dart';
+import 'package:logging/logging.dart';
+import 'package:angular/src/core/linker/app_view_utils.dart' as ng;
+
+/// Statics needed for Angular Component.
```

Comment:
nit: `//`, otherwise the doc comment only applies to the first static.

Reply:
Done